django/core/handlers/base.py

```diff
@@ -9,12 +9,16 @@
 from django.core import signals
 from django.core.exceptions import ImproperlyConfigured, MiddlewareNotUsed
 from django.db import connections, transaction
-from django.middleware.exception import ExceptionMiddleware
 from django.urls import get_resolver, get_urlconf, set_urlconf
 from django.utils import six
 from django.utils.deprecation import RemovedInDjango20Warning
 from django.utils.module_loading import import_string
-from django.views import debug
+
+from .exception import (
+    convert_exception_to_response,
```

Comment:
I see. I don't like a style that mixes single-line and multi-line -- if a list doesn't fit all in a single line, I think it's better if each item gets its own line, even if two or three could fit on a line.

But consistency and having an automated tool to enforce style are more important than my preference, so I'll change it :-)